ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -85,14 +88,27 @@ sub Configure_Cache {
     my $new_cache       = 0;     # in 1k blocks.
     my $file_afsmount   = '';
 
-    if ( $config->elementExists("$PREFIX/cachesize") ) {
-        $new_cache = $config->getValue("$PREFIX/cachesize");    #new cache size
-    }
-    else {
-        $self->info("Cannot get CDB $PREFIX/cachesize - not setting cache size");
+    my $afs_config = $config->getElement($PREFIX)->getTree();
```

Comment:
even better would be `$config->getTree($self->prefix())`, but maybe keep it in mind for a next time :smile: